prettier.config.js

```diff
@@ -1 +1,6 @@
-module.exports = require('@chanzuckerberg/prettier-config-edu');
+module.exports = {
+  arrowParens: 'always',
+  bracketSpacing: false,
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
Recreated @chanzuckerberg/prettier-config-edu here, without the tailwind plugin.

This repo is on the public (frontend-libs) version of the package, which is not compatible with Prettier 3.